pom.xml

```diff
@@ -108,10 +108,10 @@
     <avro.version>1.8.2</avro.version>
     <bcprov-jdk15on.version>1.64</bcprov-jdk15on.version>
     <calcite.version>1.25.0</calcite.version>
-    <datanucleus-api-jdo.version>5.2.4</datanucleus-api-jdo.version>
-    <datanucleus-core.version>5.2.4</datanucleus-core.version>
+    <datanucleus-api-jdo.version>5.2.6</datanucleus-api-jdo.version>
```

Comment:
According to the release notes, this memory leak issue is fixed in 5.2.7 and NOT in 5.2.6. 
https://www.datanucleus.org/documentation/news/access_platform_5_2.html#_datanucleus_accessplatform_5_2_7
So this PR will not resolve the issue.

Also there are some important bug fixes like support for sequences in MS SQL etc in later minor releases. We should atleast try to move up to 5.2.10 (5.2.11 is also available, but not many fixes that directly affect hive).

Reply:
@nrg4878 
This was my oversight, now I have upgraded the various packages to the latest version.

[datanucleus-core](https://mvnrepository.com/artifact/org.datanucleus/datanucleus-core)
[datanucleus-api-jdo](https://mvnrepository.com/artifact/org.datanucleus/datanucleus-api-jdo)
[datanucleus-rdbms](https://mvnrepository.com/artifact/org.datanucleus/datanucleus-rdbms)

@pvary @kgyrtkirk Please review again.
thanks all guys~

Reply:
Thanks @zzzzming95  for addressing the issue.